build.gradle

```diff
@@ -97,6 +97,7 @@ android {
     compileSdkVersion 26
 
     defaultConfig {
+        minSdkVersion 14
         manifestPlaceholders.versionCode = "20900300"
```

Comment:
You have removed targetSdkVersion from AndroidManifest.xml but not included it in here.